templates/search-result.tmpl

```diff
@@ -136,12 +136,24 @@
     
     <div class="navigation">
       
-      {if $previous >= 0}
+    {if $previous >= 0}
 	<span class="txtPrev">
-	  <a href="#" onclick="javascript:jumpto({$previous}); return false;">&laquo; Previous</a>
+	  <a href="#" onclick="javascript:jumpto({$previous}); return false;">&laquo;&nbsp;Previous</a>
 	</span>
 	{/if}
-
+    {for $f in range(max(0, floor($previous/$limit) - 4), 
```

Comment:
Ah, OK.  Please make sure these are also parameterized; it's best to avoid these hardcoded constants.